addons/im_livechat/static/src/embed/external/boot.js

```diff
@@ -10,12 +10,16 @@ import { Deferred } from "@web/core/utils/concurrency";
 import { registry } from "@web/core/registry";
 import { makeEnv, startServices } from "@web/env";
 import { session } from "@web/session";
+import { loadBundle } from "@web/core/assets";
 
 odoo.livechatReady = new Deferred();
 
 (async function boot() {
     session.origin = session.livechatData.serverUrl;
     await whenReady();
+    if (session.test_mode) {
+        await loadBundle("im_livechat.assets_livechat_support_tours");
```

Comment:
note: cannot be done in the template directly as odoo bundle is loaded with a defer script.